tools/lint/examples/sm-main.yang

```diff
@@ -0,0 +1,32 @@
+module sm-main {
```

Comment:
When running the command:
`../../../build/yanglint -f tree sm-main.yang`
I am getting an error:
`libyang err : Extension plugin "libyang 2 - Schema Mount, version 1": Failed to get extension data, no callback set.`

Reply:
This is fixed, now.